doc/releases/release-notes-2.0.rst

```diff
@@ -21,17 +22,18 @@ Kernel
 Architectures
 *************
 
-* TBD
+* posix: Fix race condition with terminated threads which had never been
```

Comment:
I'm holding off on recommending changes such as ``POSIX`` vs. ``posix`` here, and some grammar issues we can fix after we get more content in the release notes.